src/Type/Object/User.php

```diff
@@ -1,20 +1,10 @@
 <?php
 
-
 namespace WPGraphQL\Type\Object;
 
 use WPGraphQL\Data\DataSource;
 
-/**
- * Class User
- *
- * @package WPGraphQL\Type\Object
- */
 class User {
-
```

Comment:
why are these docblocks are being removed?

Reply:
That's weird. Will fix it.